tools/data_exploration/link_between_var.xml

```diff
@@ -0,0 +1,333 @@
+              'FALSE'
+              ''
+              '$method.columns'
+              ''
+              ''
+              ''
+            #elif $method.type == 'interr':
+              'FALSE'
+              'FALSE'
+              'FALSE'
+              'TRUE'
+              'FALSE'
+              '$method.species'
+              ''
+              '$method.variable'
+              '$method.variable2'
+              '$method.variable3'
+            #else:
+              'FALSE'
+              'FALSE'
+              'FALSE'
+              'FALSE'
+              'TRUE'
+              ''
+              ''
+              '$method.variable'
+              ''
+              ''
+            #end if
+        ]]>
+        </command>
+    <inputs>
+        <expand macro="explo_input"/>
+        <conditional name="method">
+            <param name="type" type="select" label="Variables links exploration">
+                <option value="collinearity">Collinearity between selected numerical variables for each species</option>
+                <option value="vif">Variance inflation factor (vif) on selected numerical variables</option>
+                <option value="pca">Principal component analysis (pca) on selected numerical variables</option>
+                <option value="interr">Interactions between 2 selected numerical variables</option>
+                <option value="autocorr">Autocorrelation of one selected numerical variable</option>
+            </param>
+            <when value="collinearity">
+                <param name="species" type="data_column" data_ref="input" numerical="False" label="Select column containing species for collinearity"/>
+                <param name="columns" type="data_column" data_ref="input" numerical="True" multiple="True" label="Select columns containing numerical values"/>
+            </when>
+            <when value="vif">
+                <param name="columns" type="data_column" data_ref="input" numerical="True" multiple="True" label="Select columns containing numerical values"/>
+            </when>
+            <when value="pca">
+                <param name="columns" type="data_column" data_ref="input" numerical="True" multiple="True" label="Select columns containing numerical values"/>
+            </when>
+            <when value="interr">
+                <param name="variable" type="data_column" data_ref="input" numerical="True" label="Select column containing numerical values"/>
+                <param name="variable2" type="data_column" data_ref="input" numerical="True" label="Select column containing numerical values"/>
+                <param name="species" type="data_column" data_ref="input" numerical="False" label="Select column containing species" help="This parameter allows you to divide your scatterplot according to species"/>
+                <param name="variable3" type="data_column" data_ref="input" label="Select column" help="This parameter allows you to divide your scatterplot once more"/>
```

Comment:
Maybe make these two parameters optional ? Or is it always necessary to divide the plot ?